.env.example

```diff
@@ -1,4 +1,25 @@
+# Cypress Environment Variables
+CLEVER_USERNAME=your-clever-username
+CLEVER_PASSWORD=your-clever-password
+SUPER_ADMIN_USERNAME=your-super-admin-username
+SUPER_ADMIN_PASSWORD=your-super-admin-password
+SUPER_ADMIN_ID=your-super-admin-firestore-id
+PARTNER_ADMIN_USERNAME=your-partner-admin-username
+PARTNER_ADMIN_PASSWORD=your-partner-admin-password
+PARTNER_ADMIN_ID=your-partner-admin-firestore-id
+PARTICIPANT_USERNAME=your-participant-username
+PARTICIPANT_PASSWORD=your-participant-password
+PARTICIPANT_EMAIL=your-participant-email
+PARTICIPANT_EMAIL_PASSWORD=your-participant-email-password
+
```

Comment:
```suggestion
```

Reply:
This is a good idea. I will make a note of it on ClickUp and we will add it to the tech-debt pile.

Reply:
Sorry, should have been clearer – those no longer need to be added here / should be removed as they are available in `.env.example.test` that was introduced in PR #767